### PR TITLE
Added stone variants to pathblocks

### PR DESCRIPTION
**Pull request by anonymous563**
Added stone, andesite, diorite, and granite, polished versions, and slabs and stairs for each.
https://discord.com/channels/139070364159311872/240841248750043137/833368500754972684

Review please and thanks!

### CODE REVIEW

File: src/main/resources/data/minecolonies/tags/blocks/pathblocks.json [hunk 1]
```diff
@@ -6,6 +6,27 @@
     "minecraft:stone_brick_stairs",
     "minecraft:stone_brick_slab",
     "minecraft:mossy_stone_brick_slab",
+    "minecraft:stone",
```
Comment (ravenbuilder934) [inline src/main/resources/data/minecolonies/tags/blocks/pathblocks.json]: I'd say not stone, in case there's exposed stone as part of worldgen

Comment (ravenbuilder934) [reply src/main/resources/data/minecolonies/tags/blocks/pathblocks.json]: same for andesite, diorite, and granite (the non-polished variants)

**Review by ravenbuilder934 — DISMISSED** [review]

**Review by marchermans — CHANGES_REQUESTED** [review]
This should be done by including the stone tag

**Comment by ravenbuilder934** [comment]
there is no `stone` tag according to the MC wiki, I don't know what you mean

**Comment by uecasm** [comment]
There's a `forge/stone` block tag.  But you shouldn't use it since it includes the unpolished blocks.

**Review by Mekle001 — DISMISSED** [review]

File: src/main/resources/data/minecolonies/tags/blocks/pathblocks.json [hunk 2]
```diff
@@ -6,6 +6,17 @@
     "minecraft:stone_brick_stairs",
     "minecraft:stone_brick_slab",
     "minecraft:mossy_stone_brick_slab",
+    "minecraft:stone_slab",
+    "minecraft:stone_stairs",
```
Comment (someaddons) [inline src/main/resources/data/minecolonies/tags/blocks/pathblocks.json]: I think we should only have the stone brick variants, not the plain stone

Comment (Raycoms) [reply src/main/resources/data/minecolonies/tags/blocks/pathblocks.json]: Yeah, the others are fine, let's remove those here then :shipit:

**Pull request merged**